src/main/scala/org/openeo/opensearch/backends/CreodiasClient.scala

```diff
@@ -76,6 +77,7 @@ object CreodiasClient extends OpenSearchClient {
   override def getCollections(correlationId: String): Seq[Feature] = {
     val getCollections = http(collections)
       .option(HttpOptions.followRedirects(true))
+      .timeout(connTimeoutMs = 10000, readTimeoutMs = 40000)
```

Comment:
This is already covered by the change in OpenSearchClient.

Reply:
fixed